wordpress-mobile/WordPress-Android
# [Reader Improvements] Post Details header avatar borders

Fixes #19272

## Demo

Light mode | Dark mode
--- | ---
<img src="https://github.com/wordpress-mobile/WordPress-Android/assets/5091503/e85c4c72-6d7d-42df-bb96-85d0f18d2a1d" width="300" /> | <img src="https://github.com/wordpress-mobile/WordPress-Android/assets/5091503/8d669c25-4ba6-48d0-9bfc-9b7c8b250651" width="300" />

## To test
1. Install and log into Jetpack
2. Make sure `ReaderImprovementsFeatureConfig` is enabled
3. Go to Reader
4. Open any Post
5. **Verify** the avatars have a "background-colored" outside border and background
6. **Verify** the avatars have an "on-background-colored" inside border (this is a bit harder to see because that border only has 10% opacity)

## Regression Notes
1. Potential unintended areas of impact
Borders appearing on other avatars

7. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

8. What automated tests I added (or what prevented me from doing so)
N/A, just UI.

PR submission checklist:


UI Changes testing checklist:
